### PR TITLE
[feat] Add home page with recent events component

### LINKED ISSUE #11: Vue.js Homepage Layout

### PR DESCRIPTION
**Pull request by QCrow**
Closes #11

### CODE REVIEW

File: src/pages/HomePage.vue [hunk 1]
```diff
@@ -0,0 +1,241 @@
+<template>
+  <div class="page">
+    <div class="vertical-flex-container banner-background">
+      <NavBar />
+      <div class="banner-title">
+        <div>
+          <p style="margin-left: -5vw; font-size: 7vw">MCGILL</p>
+          <div class="title-text">
```
Comment (PeiniCheng) [inline src/pages/HomePage.vue:8]: This font looks wired on Android phone

File: src/pages/HomePage.vue [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    <RecentEvents />
+
+    <div class="about-us">
+      <div id="about-us-left">
+        <img
+          id="about-us-image"
+          src="https://res.cloudinary.com/mcgillcssa/image/upload/s--ztjHHMAh--/q_jpegmini/v1574799607/cssa_jpqr5f.jpg"
+        />
+        <div class="vertical-flex-container">
+          <div id="about-us-description">
```
Comment (PeiniCheng) [inline src/pages/HomePage.vue:26]: The about us text is not fully displayed on the smaller window

**Review by PeiniCheng — CHANGES_REQUESTED** [review]
Looks so cool! Font issue are not important but the aboutus part might need to be fixed I think

File: src/pages/HomePage.vue (hunk 2 above)
Comment (QCrow) [reply src/pages/HomePage.vue:26]: Added a quick fix that changes the layout from row to column on smaller screens. I will modify it to find a better solution later after the media design team verifies the page.

**Review by PeiniCheng — APPROVED** [review]
Awesome

**Pull request merged**